### PR TITLE
DIGI_14_CreateTaxParameter

### PR DESCRIPTION
**Pull request by prakashkolli**
1. created TaxParameter Controller
2.created create View
3.changes done on header menu
4.created Tax Parameter Library
5.created EndPoint
6.Created methods in Library Utility
7.created taxParameter Model

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                Task<HttpResponseMessage> res = taxParameterLibrary.CreateTaxParameter(Taxmodel);
+               if(res.IsFaulted)
+               {
+                    TempData["fmessage"] = "Failed To Insert Record";
+                    return RedirectToAction("Create");
+               }
+            }
+            catch (Exception ex)
+            {
+                ViewBag.ErrorMessage = "";
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs:41]: add view bag message

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Shared/Partial/_HeaderMenu.cshtml [hunk 2]
```diff
@@ -59,13 +59,14 @@
...
                     <li class="dropdown">
-                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Paymetns Info<b class="caret"></b></a>
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Payments Info<b class="caret"></b></a>
                         <ul class="dropdown-menu multi-column columns-3">
                             <div class="row">
                                 <div class="multi-gd-img">
                                     <ul class="multi-column-dropdown">
                                         <li><a href="personalcare.html">Order Payments</a></li>
                                         <li><a href="personalcare.html">Cancellations</a></li>
+                                        <li><a href="~/TaxParameter">Tax Parameters</a></li>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/Shared/Partial/_HeaderMenu.cshtml:69]: You have commeted this file with list taxpage parameter.,please revert this file

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs [hunk 3]
```diff
@@ -13,49 +13,18 @@ namespace DigiOrderEnterpriseLibrary
...
         /// Date: 7/27/2020
         /// Description:To post the data with Token Authentication
         /// </summary>
         public async Task<HttpResponseMessage> POSTAsync(string endPoint, string Json)
         {
-            Task<HttpResponseMessage> response = null;
+            
             try
             {
                 using (var client = new HttpClient())
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs:30]: Doesnt  do any changes in libraryutility .keep as per devbranch only

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/TaxParameterModel.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+namespace DigiOrderEnterpriseModel
+{
+    public class TaxParameterModel
+    {
+        public int SNO { get; set; }
+        public Guid GUID { get; set; }
+        [Required]
+        public string TITLE { get; set; }
+        [Required]
+        public string DESCRIPTION { get; set; }
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/TaxParameterModel.cs:14]: you have commeted this file with list all taxparameter.Revert this file

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            ViewBag.ControlNviation = "Create TaxParameter";
+            return View(new TaxParameterModel());
+        }
+
+        /// Author: Prakash Kolli
+        /// Date: 7/1/2020
+        /// Description: Insert A TaxParameter TO Database
+        /// Ref. StoryId: DIGI 14
+        /// Reviewer:
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs:25]: add reviewer name

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs (hunk 1 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs]: changed

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs (hunk 3 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs]: changed as per dev branch

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs (hunk 5 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs]: added

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 6]
```diff
@@ -250,6 +251,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="App_Data\" />
+    <Folder Include="Core\" />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:254]: Remove  line 254

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Shared/Partial/_HeaderMenu.cshtml (hunk 2 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterprise/Views/Shared/Partial/_HeaderMenu.cshtml]: reverted

**Pull request merged**